cpp/ql/src/Likely Bugs/JapaneseEra/ConstructorOrMethodWithExactEraDate.ql

```diff
@@ -1,12 +1,14 @@
 /**
- * @name Hard-coded Japanese era start date
+ * @name Hard-coded Japanese era start date in call
  * @description Japanese era changes can lead to code behaving differently. Avoid hard-coding Japanese era start dates.
  * @kind problem
  * @problem.severity warning
  * @id cpp/japanese-era/constructor-or-method-with-exact-era-date
  * @precision medium
  * @tags reliability
  *       japanese-era
+ * @deprecated This query is deprecated, use
+ *             Best Practices/Magic Constants/JapaneseEraDate.ql instead.
```

Comment:
I think the modern way we refer to queries is by `@id`.

Reply:
Good point.  I will update this and the place I copied it from.

Reply:
Done.